test/coverage.jl

```diff
@@ -436,4 +441,23 @@ end # of automaton
         end) == 1
     end
 
+    @testset "Some other normally unreachable code 1" begin
+        f = Rematch2.BoundFalsePattern(LineNumberNode(@__LINE__, @__FILE__), false)
+        @test_throws ErrorException Rematch2.next_action(f)
+
+        node = Rematch2.AutomatonNode(Rematch2.CasePartialResult[])
+        pattern = Trash(LineNumberNode(@__LINE__, @__FILE__))
+        binder = Rematch2.BinderContext(@__MODULE__)
+        @test_throws ErrorException Rematch2.make_next(node, pattern, binder)
+    end
+
+    @testset "Abstract types" begin
+        x = 2
+        @test (@match x begin
+            ::Abstract1 => 1
```

Comment:
this abstract type has no concrete types, but if the value matched were a subtype, would this work?

Reply:
Yes, it would.  There are tests for that elsewhere.  This is testing what happens when you have two unrelated abstract types.